docs/lti-integration.md

```diff
@@ -0,0 +1,141 @@
+# LTI 1.3 Integration
+
+We are working to fully support LTI 1.3 as an LTI Tool. Currently, we are beta
+testing with a small group of Canvas users, and plan to release phases of LTI 1.3
+features in 2024.
+
+Below are instructions on how to register Code.org as an LTI Tool in Canvas.
+NOTE: this pilot is currently supported for a closed group, and is not get
+generally available.
+
```

Comment:
I think we should include a high-level set of steps/walkthroughs to guide teachers before introducing them to the specifics. E.g. per our discussions:

To install Code.org as an LTI Integration, we will need a Developer Key from your Canvas instance. Once the Developer Key is obtained, we will <....> etc. etc.

Reply:
Rather than copying over Canvas instructions, I'm hoping to have the user reference the Canvas set up instructions that are linked below. This allows us to keep Canvas docs as the source of truth, and helps mitigate drift between their docs and our copy. Thoughts?

Reply:
This process also shouldn't be new for LMS admins. This is a flow they would use to install all LTI Tools in Canvas.

Reply:
If we don't copy over the relevant pieces, I think we should at least be precise about which steps they need to reference. Teachers are busy & overworked, if the documentation is complex or has too many drop-off opportunities, they may not install the integration

Reply:
They have to follow all the steps. There are no shortcuts. This is standard LTI Tool configuration for Canvas.